example/build.gradle

```diff
@@ -0,0 +1,29 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath fileTree("../plugin/build/libs")
+    }
+}
+
+plugins {
+    id "com.github.hierynomus.license" version "0.12.1"
```

Comment:
So in this case you cannot as the example application is not linked with the root build.gradle file. This was on purpose as it allows the plugin to be built without evaluating the existence of the plugin for the example project.